libs/live-config/src/featureFlags/firebaseFeatureFlags.ts

```diff
@@ -3,13 +3,13 @@ import { snakeCase } from "lodash";
 import semver from "semver";
 import { Feature, FeatureId } from "@ledgerhq/types-live";
 import { getEnv } from "@ledgerhq/live-env";
-import { LiveConfig } from "./LiveConfig";
+import { LiveConfig } from "../LiveConfig";
 
 export type FirebaseFeatureFlagsProviderProps = PropsWithChildren<unknown>;
 export const formatToFirebaseFeatureId = (id: string) => `feature_${snakeCase(id)}`;
 
 export const checkFeatureFlagVersion = (feature: Feature) => {
-  const platform = LiveConfig.getInstance().platform;
+  const platform = LiveConfig.configInstance!.platform;
```

Comment:
I don't think the ! is needed on `configInstance!` you should remove it if possible (also on other places in this file)

Reply:
good point. Fixed.